mscore/scoretab.h

```diff
@@ -32,9 +32,9 @@ enum class MagIdx : char;
 //---------------------------------------------------------
 
 struct TabScoreView {
-      Score* score;
+      MasterScore* score;
```

Comment:
Why did you change this, whether `TabScoreView` now operates on `MasterScore's` only? Who is responsible for updating excerpts views?

Reply:
In fact, every score that was stored in this structure was a `MasterScore` since [the only place this structure's instances were created](https://github.com/musescore/MuseScore/blob/4899a7cd69cd6594351e10114ef039d6bb79707a/mscore/scoretab.cpp#L352) is an `ScoreTab::insertTab` function which accepts `MasterScore`s only. I changed this line to further ensure it and avoid type casts when [using `MasterScore::excerptsChanged`](https://github.com/musescore/MuseScore/pull/4004/files#diff-1baba45712fb8c496822b81e84988e48R268) function on the stored score.
Excerpt views are handled by `ScoreTab` either and information on the current excerpt view is represented by `part` integer field in the discussed `TabScoreView` structure.